### ISSUE TITLE
Use libblas as the default BLAS libaray

### ISSUE THREAD

**Issue by gxm11** [crystal-data/num.cr#66]
In the top of file `src/libs/cblas.cr` it shows the default BLAS libaray is `openblas`. 

```crystal
{% if flag?(:openblas) %}
  @[Link("openblas")]
{% elsif flag?(:accelerate) %}
  @[Link(framework: "Accelerate")]
{% elsif flag?(:darwin) %}
  @[Link(framework: "Accelerate")]
{% else %}
  @[Link("openblas")] # => change to @[Link{"blas"}]
{% end %}
```

Could you change the default library to `@[Link{"blas"}]`? Or, add a flag to make the link to `libblas`?

 It will be helpful for some system only `libblas` is installed (like crystal env of replit.com).

**Comment by christopherzimmerman** [comment]
I would accept a PR making that change. I think anyone who is concerned with performance would know to use openblas and those that just want to test the library would benefit from having it usually work out of the box.

**Comment by christopherzimmerman** [comment]
Closed in #67
